content/metadata.yaml

```
@@ -1541,6 +1541,10 @@ authors:
         order: 6
         contributions:
           - Writing - Review & Editing
+      vaccines-novel:
+	order: 7
```

```suggestion
        order: 7
```
This tab character is causing the build failure